Resources/Prototypes/DeltaV/Entities/Structures/Storage/SecureCabinet/secure_cabinets.yml

```diff
@@ -0,0 +1,319 @@
+#Parent
+- type: entity
+  parent: [ BaseStructureDynamic, BaseBagOpenClose ]
+  id: BaseSecureCabinet
+  description: A secure cabinet to keep all your confidential files or items away from prying hands.
+  suffix: Empty
+  categories: [ HideSpawnMenu ]
+  abstract: true
+  components:
+  - type: Storage
+    grid:
+    - 0,0,2,1
+    - 0,3,2,4
+    - 0,6,2,7
+    maxItemSize: Normal
+  - type: Sprite
+    sprite: DeltaV/Structures/Storage/secure_cabinet.rsi
+    noRot: true
+  - type: Appearance
+  - type: UserInterface
+    interfaces:
+      enum.StorageUiKey.Key:
+        type: StorageBoundUserInterface
+  - type: Transform
+    noRot: true
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.22,-0.40,0.22,0.40"
+        density: 200
+        mask:
+        - MachineMask
+        layer:
+        - MachineLayer
+  - type: InteractionOutline
+  - type: ContainerContainer
+    containers:
+      storagebase: !type:Container
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 50
+      behaviors:
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+      - !type:PlaySoundBehavior
+        sound:
+          collection: MetalBreak
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          SheetSteel1:
+            min: 1
+            max: 2
+  - type: StaticPrice
+    price: 80
+  - type: Construction
+    graph: SecureCabinet
+  
+#Empty Secure Cabinet
+- type: entity
+  name: Secure Cabinet
```

Comment:
Nitpick, but if you can make it follow [conventions](https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html#yaml-conventions) but TL;DR: the order should be `type > abstract > parent > id > name > description > components`